_data/simple-icons.json

```diff
@@ -1548,6 +1548,11 @@
         {
             "title": "JET",
             "hex": "FBBA00",
+            "source": "https://www.jetbrains.com/company/brand/"
+        },
+        {
+            "title": "JetBrains",
+            "hex": "000000",
             "source": "https://de.wikipedia.org/wiki/Datei:JET.svg"
```

Comment:
Seems like something went wrong here :wink: Note that the source of the JetBrains should go together with the rest of the object (i.e. the part between `{` and `}`) and the source of "JET" should stay unchanged.

You can click "Commit suggestions" in the comments below to fix this :+1:

Reply:
Thanks for the fixes @ulasaydin 🙂